### PR TITLE
add latest fifa with new schema

### PR DESCRIPTION
**Pull request by James-Sangalli**
For #3 

@colourful-land you might wish to sign this with another key?

### CODE REVIEW

**Comment by James-Sangalli** [comment]
@colourful-land this has been updated again and is ready to merge.

File: shong.wang/2019/05/fifa.tsml [hunk 1]
```diff
@@ -0,0 +1,994 @@
...
+
+    <ts:attribute-type id="time" syntax="1.3.6.1.4.1.1466.115.121.1.24">
+        <ts:name>
+            <ts:string xml:lang="en">Time</ts:string>
+            <ts:string xml:lang="zh">时间</ts:string>
+            <ts:string xml:lang="es">Tiempo</ts:string>
+            <ts:string xml:lang="ru">время</ts:string>
+        </ts:name>
+        <ts:origins>
+            <ts:token-id as="utf8" bitmask="000000000000000000000000000000000000FFFFFFFF00000000000000000000"/>
```
Comment (hboon) [inline shong.wang/2019/05/fifa.tsml]: This should be `as="uint"` instead:

```
<ts:token-id as="utf8" bitmask="000000000000000000000000000000000000FFFFFFFF00000000000000000000"/>
```

And we need a mapping for the uint to generalised time.

**Comment by James-Sangalli** [comment]
@hboon fifa uses binary time, added this amendment and converted to uint

**Comment by colourful-land** [comment]
> @hboon fifa uses binary time, added this amendment and converted to uint

That's not how I remembered it. FIFA's time is timezone sensitive, e.g. a match at 18.00 doesn't show up as 15.00 if you are in England. If you use BinaryTime, it will show as 15.00 if the user is in England.

The way I remembered it, it is a mapping to GeneralisedTime. The attribute type is the type of data after mapping, not before mapping.

**Comment by James-Sangalli** [comment]
@colourful-land @hboon fixed with generalised time mapping

**Comment by colourful-land** [comment]
@James-Sangalli your file doesn't validate. Look out for the `'as'` keyword in the error:
````
$ JVMOPTS=-Djavax.xml.accessExternalDTD=all /opt/xmlsectool-2.0.0/xmlsectool.sh --validateSchema --xsd --schemaDirectory ~/IdeaProjects/TokenScript/schema/tokenscript.xsd --inFile EntryToken.tsml 
INFO  XMLSecTool - Reading XML document from file 'EntryToken.tsml'
INFO  XMLSecTool - XML document parsed and is well-formed.
ERROR XMLSecTool - XML is not schema valid
org.xml.sax.SAXParseException: cvc-complex-type.3.2.2: No está permitido que el atributo 'as' aparezca en el elemento 'ts:attribute-type'.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:134) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:396) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator$XSIErrorReporter.reportError(XMLSchemaValidator.java:453) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.reportSchemaError(XMLSchemaValidator.java:3231) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.processAttributes(XMLSchemaValidator.java:2708) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:2051) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:741) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.jaxp.validation.DOMValidatorHelper.beginNode(DOMValidatorHelper.java:277) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.jaxp.validation.DOMValidatorHelper.validate(DOMValidatorHelper.java:244) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.jaxp.validation.DOMValidatorHelper.validate(DOMValidatorHelper.java:190) ~[na:1.8.0_191]
	at com.sun.org.apache.xerces.internal.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:109) ~[na:1.8.0_191]
	at javax.xml.validation.Validator.validate(Validator.java:124) ~[na:1.8.0_191]
	at net.shibboleth.tool.xmlsectool.SchemaValidator.validate(SchemaValidator.java:146) ~[xmlsectool-2.0.0.jar:na]
	at net.shibboleth.tool.xmlsectool.XMLSecTool.schemaValidate(XMLSecTool.java:342) [xmlsectool-2.0.0.jar:na]
	at net.shibboleth.tool.xmlsectool.XMLSecTool.main(XMLSecTool.java:144) [xmlsectool-2.0.0.jar:na]
````

**Comment by James-Sangalli** [comment]
> @James-Sangalli you signed with the wrong key. should use sktravel.cn key. I have the right key (albeit expired) but can you point me to the unsigned XML for me to sign a new version?

https://github.com/AlphaWallet/TokenScript/tree/fifa-DO-NOT-MERGE

**Pull request merged**